src/registrar/forms/application_wizard.py

```diff
@@ -180,10 +180,10 @@ def clean(self):
             or self.cleaned_data["state_recognized_tribe"]
         ):
             raise forms.ValidationError(
-                "Only tribes recognized by the U.S. federal government or by a U.S."
-                " state government are eligible for .gov domains. Please email"
-                " registrar@dotgov.gov to tell us more about your tribe and why you"
-                " want a .gov domain.",
+                "Only tribes recognized by the U.S. federal government
+                " or by a U.S. state government are eligible for .gov
+                " domains. Please <a href="{% url 'todo' %}">tell us more about your tribe and why
```

Comment:
**comment (non-blocking)**: This might not look right on the page because Django might escape the HTML codes from the error message specified here. `mark_safe` is how we've dealt with this in other similar situations and I'm happy to help work it out.

@michelle-rago How does this look in the sandbox?

Reply:
It didn't deploy to my sandbox ☹️